features/command_line_interface/options.feature

```diff
@@ -41,7 +41,9 @@ Feature: Reek can be controlled using command-line options
 
       Configuration:
           -c, --config FILE                Read configuration options from FILE
-              --smell SMELL                Detect smell SMELL (default: all enabled smells)
+              --smell SMELL                Only look for a specific smell.
+                                           Call it like this: reek --smell PrimaDonnaMethod source.rb
+                                           Check out https://github.com/troessner/reek/blob/master/docs/Code-Smells.md for a list of smells
```

Comment:
@troessner: Maybe we start with an issue, because there are some questions that will likely crop up, like what and how much to print, and whether it needs to be reconsidered as the list of smells grows. 😀